examples/go/with_proto/proto/translators.go

```diff
@@ -0,0 +1,11 @@
+package proto
+
+import (
+	"time"
+)
+
+// Time translation functions.
+
+func ToProtoTime(t time.Time) Time {
```

Comment:
Are there instructions to manually test this?

Reply:
Added a comment with instructions.